admin/class-gutenberg-compatibility.php

```diff
@@ -0,0 +1,132 @@
+<?php
+/**
+ * WPSEO plugin file.
+ *
+ * @package WPSEO\Gutenberg_Compatibility
+ */
+
+/**
+ * Class WPSEO_Gutenberg_Compatibility
+ */
+class WPSEO_Gutenberg_Compatibility {
+
+	/**
+	 * The currently released version of Gutenberg.
+	 */
+	const CURRENT_RELEASE = '3.4.0';
+
+	/**
+	 * The minimally supported version of Gutenberg by the plugin.
+	 */
+	const MINIMUM_SUPPORTED = '2.8.0';
+
+	/**
+	 * @var string
+	 */
+	protected $current_version;
+
+	/**
+	 * WPSEO_Gutenberg_Compatibility constructor.
+	 */
+	public function __construct() {
+		$this->current_version = $this->detect_installed_gutenberg_version();
+	}
+
+	/**
+	 * Determines whether or not Gutenberg is installed.
+	 *
+	 * @return bool Whether or not Gutenberg is installed.
+	 */
+	public function is_installed() {
+		return $this->is_gutenberg_available() && $this->current_version !== '';
+	}
+
+	/**
+	 * Determines whether or not the currently installed version of Gutenberg is below the minimum supported version.
+	 *
+	 * @return bool True if the currently installed version is below the minimum supported version. False otherwise.
+	 */
+	public function is_below_minimum() {
+		return $this->get_major_minor_version( $this->current_version ) < $this->get_major_minor_version( $this->get_minimum_supported_version() );
```

Comment:
Why not using `version_compare`?

Reply:
We decided not to do this, as `version_compare` also takes patch versions into account and that's not something we want to do.